Plugins/Flow.Launcher.Plugin.ProcessKiller/ProcessHelper.cs

```diff
@@ -80,6 +93,26 @@ public void TryKill(Process p)
             catch (Exception e)
             {
                 Log.Exception($"|ProcessKiller.CreateResultsFromProcesses|Failed to kill process {p.ProcessName}", e);
+                TryKillRunAs(p);
+            }
+        }
+
+        public void TryKillRunAs(Process p)
+        {
+            try
+            {
+                Process process = new Process();
+                ProcessStartInfo startInfo = new ProcessStartInfo();
+                startInfo.WindowStyle = ProcessWindowStyle.Hidden;
+                startInfo.FileName = "powershell.exe";
+                startInfo.Arguments = $"Start cmd.exe -ArgumentList \"/k\",\"taskkill\",\"/f\",\"/pid\", \"{p.Id}\" -Verb Runas";
```

Comment:
I forget this one. It's pretty weird that you start `taskkill` inside a `cmd` inside the `powershell`.

Reply:
The startInfo do provide an `ArgumentList` API that worth taking a look.